src/js/extensions/anchor-preview.js

```diff
@@ -119,12 +121,41 @@
             }
             defaultLeft = diffLeft - halfOffsetWidth;
 
-            this.anchorPreview.style.top = Math.round(buttonHeight + boundary.bottom - diffTop + this.window.pageYOffset - this.anchorPreview.offsetHeight) + 'px';
+            // If container element is absolute / fixed, recalculate boundaries to be relative to the container
+            if (elementsContainerAbsolute) {
+                elementsContainerBoundary = elementsContainer.getBoundingClientRect();
+                ['top', 'left'].forEach(function (key) {
+                    relativeBoundary[key] = boundary[key] - elementsContainerBoundary[key];
+                });
+
+                ['bottom', 'right'].forEach(function (key) {
+                    relativeBoundary[key] = elementsContainerBoundary[key] - boundary[key];
+                });
+
+                relativeBoundary.width = boundary.width;
+                relativeBoundary.height = boundary.height;
+                boundary = relativeBoundary;
+
+                containerWidth = elementsContainerBoundary.width;
+            }
+
+            middleBoundary = boundary.left + boundary.width / 2;
+            top = buttonHeight + boundary.top + boundary.height - diffTop - this.anchorPreview.offsetHeight;
+
+            // If container element is absolute / fixed, adjust top position according to container scroll position
```

Comment:
Same feedback here as in toolbar.js, could we get rid of this extra `if()` block?